src/Analyzers/CSharp/Tests/RemoveUnnecessaryParentheses/RemoveUnnecessaryExpressionParenthesesTests.cs

```diff
@@ -1,4 +1,4 @@
-// Licensed to the .NET Foundation under one or more agreements.
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
It looks like the original file didn't have BOM character and Visual Studio added it automatically for me.

@sharwell FYI.

Reply:
Correct, the BOM was unintentionally removed in c231e0a61eeba5e14baba9e3c5acef2f17348fae.